README.md

```diff
@@ -37,9 +42,27 @@ Open Food Facts is a non-profit association of volunteers.
 ### For everyone
 
 Data about food is of public interest and has to be open (i.e available to everyone). The complete database is published as open data and can be reused by anyone and for any use. Check-out the cool reuses or make your own!
+</details>
 
 * <https://world.openfoodfacts.org>
 
+## Feature Sprint 
+- We use feature-based sprints, [tracked here](https://github.com/orgs/openfoodfacts/projects/32)
+
+<details><summary><h2> Weekly meetings </h2></summary>
```

Comment:
This part does not render correctly.

![image](https://user-images.githubusercontent.com/8158668/208091204-2dafa47c-b095-4ef5-adf5-5d8680e26597.png)

Reply:
Je vais enlever le collapse voir si ça règle le souci

Reply:
works now, was a missing carriage return